.github/workflows/ci.yml

```diff
@@ -29,7 +29,6 @@ jobs:
           - "2.7"
           - "2.6"
           - "2.5"
-          - "2.4"
```

Comment:
@timriley are we officially dropping support for 2.4? if yes, then it shouldn't be part of this PR

Reply:
@solnic I had to do this because hanami-utils unstable branch would no longer install on Ruby 2.4, so that particular build would fail.

@jodosha — what's your perspective on how to sort this out? And what's our Ruby version support for these unstable/eventually-2.0 branches?

Reply:
For those following this PR: we've sorted this out. Dropping 2.4 (at least) from unstable branches of gems.